btr-api/src/btr_api/models/base.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy.sql import func
+from sqlalchemy import Column, Integer, DateTime
+
+from .db import db
+
+
+class BtrModelBase:
+    id = db.Column(db.Integer, primary_key=True)
+
+    created_by = Column(Integer)
+    updated_by = Column(Integer)
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    updated_at = Column(DateTime(timezone=True), onupdate=func.now())
```

Comment:
This sort of thing would already be tracked via the submission table right? That would be our ledger for tracking updates. Maybe instead we should add submission_id here instead or we can go without it?

Reply:
kept it, as we discussed to keep some tables separate from the submission